pandas/io/excel/_odswriter.py

```diff
@@ -99,6 +99,7 @@ def _write_cells(
         startrow: int = 0,
         startcol: int = 0,
         freeze_panes: tuple[int, int] | None = None,
+        autofilter_range: str | None = None,
```

Comment:
It seems to me we should be raising an error here if a user specifies `autofilter=True` and it is not supported with an engine rather than silently ignore the user's specification.